src/TheCallCenter/Startup.cs

```diff
@@ -51,13 +47,16 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
       }
 
       app.UseStaticFiles();
+      app.UseRouting();
       app.UseCookiePolicy();
 
-      app.UseMvc(routes =>
+      app.UseEndpoints(routes =>
```

Comment:
I can't seem to find app.MapControllers. I found IEndpointRouteBuilder extension method, but that would be replacing routes.MapControllerRoute instead of app.UseEndpoints. We're still using "Endpoints"

https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.builder.controllerendpointroutebuilderextensions.mapcontrollers?view=aspnetcore-6.0

Could you please explain what you mean by this comment?